.travis.yml

```diff
@@ -1,16 +1,29 @@
+# https://travis-ci.org/sqlalchemy-redshift/sqlalchemy-redshift
+sudo: false
 language: python
-python: 3.5
+python: "3.6"
+
 env:
   matrix:
-  - TOX_ENV=py26
-  - TOX_ENV=py27
-  - TOX_ENV=py33
-  - TOX_ENV=py34
-  - TOX_ENV=py35
-  - TOX_ENV=lint
+    - TOXENV=py27
+    - TOXENV=py34
+    - TOXENV=py35
+    - TOXENV=py36
+    - TOXENV=lint
   global:
-  - secure: g/ta2yudfA55ruxd+GHxYQUd39OM/hoEhNp0KgPm3QINaXwaB6C9eQ31qOi/i9OiSjmVwISE+yUaPtQP2jtbYnkS/SHiRNvzhfA+mPrT0n0hoVSNIlxl94E8hoPBZr2jlYVQ6dIuT98mIRYJu/wiGHDQCuar6znpiKFB1/gyUszvZxAxHJIe5EcjuXWFnre91b96tNkZRKJZv+Jst0CkknVcKAZ4kmC/o5Z1hr5VaMekRV5SKf+vMEfVc+W9VktqDBgKwaFIwWqSgQFk5LOnPr/EZq8mrqPpa+LFaLiQpDGDAdaVEM0/A3dzx3YnYoBFNr5hmCtoCLWccxC+diaw3kZuTX6olxn8cbRCVf8lu4JuJKFU5CgTLS7NBH3Rpk4iRE7C9v4W+QKjw3CJxLCMoMqQCbYx6GjzjZYbZBA6jvaS6ZFe41N6egA0SpNvKtfYU9TR7lNVwGehRIY9kycOEX5ZQInkqILxz5YHrkMNfICdLY+rYFBDvgNAOZxYR1tLaOG9jYuUbBOLVRZz78wkPrXynquay6HZn7AxBxOdnvGQAkINvo40Lx/MWdZ88o3f/th838cpPCxBRQZ5Kn3RkHNhGHwUJadsbaOLnKrj3ISaTOvq0brA1mxJFyJwoVrn+rFs80CWRonQ5lQeaOHf0p1flH7j+TFhl/INkvXsf7Y=  # PGPASSWORD
+    - secure: g/ta2yudfA55ruxd+GHxYQUd39OM/hoEhNp0KgPm3QINaXwaB6C9eQ31qOi/i9OiSjmVwISE+yUaPtQP2jtbYnkS/SHiRNvzhfA+mPrT0n0hoVSNIlxl94E8hoPBZr2jlYVQ6dIuT98mIRYJu/wiGHDQCuar6znpiKFB1/gyUszvZxAxHJIe5EcjuXWFnre91b96tNkZRKJZv+Jst0CkknVcKAZ4kmC/o5Z1hr5VaMekRV5SKf+vMEfVc+W9VktqDBgKwaFIwWqSgQFk5LOnPr/EZq8mrqPpa+LFaLiQpDGDAdaVEM0/A3dzx3YnYoBFNr5hmCtoCLWccxC+diaw3kZuTX6olxn8cbRCVf8lu4JuJKFU5CgTLS7NBH3Rpk4iRE7C9v4W+QKjw3CJxLCMoMqQCbYx6GjzjZYbZBA6jvaS6ZFe41N6egA0SpNvKtfYU9TR7lNVwGehRIY9kycOEX5ZQInkqILxz5YHrkMNfICdLY+rYFBDvgNAOZxYR1tLaOG9jYuUbBOLVRZz78wkPrXynquay6HZn7AxBxOdnvGQAkINvo40Lx/MWdZ88o3f/th838cpPCxBRQZ5Kn3RkHNhGHwUJadsbaOLnKrj3ISaTOvq0brA1mxJFyJwoVrn+rFs80CWRonQ5lQeaOHf0p1flH7j+TFhl/INkvXsf7Y=  # PGPASSWORD
+
+matrix:
+  fast_finish: true
```

Comment:
??

Reply:
https://blog.travis-ci.com/2013-11-27-fast-finishing-builds/

Reply:
We don't have any allowed failures so I'm pretty sure it's redundant

Reply:
That's fair. I copied this from my usual travis template; I can remove it but it's good practice to leave it in imo.